src/io/result_io.jl

```diff
@@ -657,43 +654,46 @@ function load_results(result_loc::String)::ResultSet
     )
 
     outcomes = Dict{Symbol,YAXArray}()
+    outcomes_properties = [:metric_name, :metric_unit, :axes_names, :axes_units]
     subdirs = filter(isdir, readdir(joinpath(result_loc, RESULTS); join=true))
     for sd in subdirs
-        if !(occursin(LOG_GRP, sd)) && !(occursin(INPUTS, sd))
-            res = zopen(sd; fill_as_missing=false)
-            sz = size(res)
-
-            # Construct dimension names and metadata
-            st = []
-            for (i, s) in enumerate(res.attrs["structure"])
-                if s == "timesteps"
-                    push!(st, input_set.attrs["timeframe"])
-                elseif s == "locations"
-                    push!(st, res.attrs["unique_loc_ids"])
-                else
-                    push!(st, 1:sz[i])
-                end
+        data = zopen(sd; fill_as_missing=false)
+        data_size = size(data)
+
+        # Construct dimension names and metadata
+        dims = []
+        for (idx, dim) in enumerate(data.attrs["structure"])
```

Comment:
I'm okay leaving this as is as it probably does not matter in this specific case, just be aware you're overwriting methods here in this scope (I'm 90% sure `dims` and `dim` are methods).

Reply:
Done